modules/dnn/CMakeLists.txt

```diff
@@ -292,8 +299,9 @@ if(TARGET ocv.3rdparty.cann AND OPENCV_TEST_DNN_CANN)
   endif()
 endif()
 
-if(HAVE_FLATBUFFERS)
+ocv_option(OPENCV_TEST_DNN_TFLITE "Build test with TFLite" (TARGET ocv.3rdparty.flatbuffers))
```

Comment:
Do we really need this option?

Reply:
Thanks! Updated handing of this option in tests. It should be more clear now (flatbuffers -> tflite)